js/layout-javascript/news-feed-code.js

```diff
@@ -1435,7 +1435,11 @@ DynamicList.prototype.renderLoopHTML = function () {
           var selector = $lineClamp.length ? '[data-line-clamp]' : '.news-feed-item-description';
           var linesToClamp = $lineClamp.length ? $lineClamp.data('line-clamp') : 3;
 
-          window.ellipsed.ellipsis(selector, linesToClamp);
+          window.ellipsed.ellipsis(selector, linesToClamp, {delimiter: '',});
+        } else {
+          var newsDescriptionSelector = '.news-feed-item-description';
+
+          $(newsDescriptionSelector).css('max-height', '4.4em');
```

Comment:
Why is this necessary?

Reply:
@tonytlwu 
While I was fixing this issue I've noticed that in IE11 text wasn't ended with ellipses, it was because of the height attribute in the CSS, so I removed it from the CSS. But height is needed for the CSS clamp that we use in other browsers except for the IE11 that is why we are adding it here.

Reply:
@YaroslavOvdii OK. If it's not related to this issue of long words, then let's keep it separate. Log a different issue and have it go through the same QA process. We've already see in this comment https://github.com/Fliplet/fliplet-studio/issues/5868#issuecomment-610783432 that mixing fixes could lead to confusions in the future.